cli/smartcontract/smart_contract.go

```diff
@@ -554,10 +557,20 @@ func invokeInternal(ctx *cli.Context, signAndPush bool) error {
 
 	if signAndPush {
 		gas = flags.Fixed8FromContext(ctx, "gas")
-		acc, err = getAccFromContext(ctx)
+		acc, wall, err = getAccFromContext(ctx)
 		if err != nil {
 			return err
 		}
+		for i := range cosigners {
+			cosignerAcc := wall.GetAccount(cosigners[i].Account)
+			if cosignerAcc == nil {
+				return cli.NewExitError(fmt.Errorf("can't calculate network fee: no account was found in the wallet for cosigner #%d", i), 1)
```

Comment:
What happens if cosigner is a contract?

Reply:
It's OK for a cosigner to be a contract. `AddNetworkFee` is able to handle this case, it will call `verify` method for this contract to define network fee, like for any other contract signer.

But still, the cosigner (contract/sig/multisig) should be presented in the wallet (and have `isDeployed` == true for the contract cosigner). Otherwise, we won't be able to calculate the network fee for the transaction. 

I'll write  a test for it.